docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -8,23 +8,33 @@ Prerequisites
 
 To use PyBaMM, you must have Python 3.8, 3.9, 3.10, 3.11, or 3.12 installed.
 
-.. tab:: Debian-based distributions (Debian, Ubuntu, Linux Mint)
+.. tab:: Debian-based distributions (Debian, Ubuntu)
 
-   To install Python 3 on Debian-based distributions (Debian, Ubuntu, Linux Mint), open a terminal and run
+   To install Python 3 on Debian-based distributions (Debian, Ubuntu), open a terminal and run
 
    .. code:: bash
 
-      sudo apt update
-      sudo apt install python3
+      sudo apt-get update \
+      sudo apt-get install python3
 
 .. tab:: Fedora/CentOS
 
    On Fedora or CentOS, you can use DNF or Yum. For example
 
    .. code:: bash
 
+      sudo dnf upgrade \
```

Comment:
Why is this "\" added here? I think the slashes should probably be removed from the instructions. It is two different commands, so if anything it should be something like `sudo dnf upgrade && sudo ....`

Usually separate lines in bash code blocks are considered different commands. I would think that we want to keep them distinct commands